src/transforms/eigenanalysis.jl

```diff
@@ -148,31 +167,52 @@ function eigenmatrices(transform, Y)
 end
 
 """
-    PCA()
+    PCA(; ndim=nothing)
```

Comment:
Let's make it a positional argument. It doesn't have any argument so there is no confusion.